runtime/src/sandbox_state.c

```diff
@@ -23,31 +23,6 @@ const char *sandbox_state_labels[SANDBOX_STATE_COUNT] = {
 	[SANDBOX_ERROR]         = "Error"
 };
 
-#ifdef LOG_SANDBOX_COUNT
-_Atomic uint32_t sandbox_state_count[SANDBOX_STATE_COUNT];
+#ifdef SANDBOX_STATE_TOTALS
+_Atomic uint32_t sandbox_state_totals[SANDBOX_STATE_COUNT];
 #endif
-
-/*
- * Function intended to be interactively run in a debugger to look at sandbox totals
- * via `call runtime_log_sandbox_states()`
- */
-void
-runtime_log_sandbox_states()
-{
-#ifdef LOG_SANDBOX_COUNT
```

Comment:
I believe that I removed this function because it was trivial to log this stuff manually with gdb commands.